stl/inc/algorithm

```diff
@@ -2622,15 +2622,15 @@ void _Pop_heap_hole_by_index(_RanIt _First, _Iter_diff_t<_RanIt> _Hole, _Iter_di
     const _Diff _Max_sequence_non_leaf = (_Bottom - 1) >> 1; // shift for codegen
     while (_Idx < _Max_sequence_non_leaf) { // move _Hole down to larger child
         _Idx = 2 * _Idx + 2;
-        if (_DEBUG_LT_PRED(_Pred, *(_First + _Idx), *(_First + (_Idx - 1)))) {
+        if (_DEBUG_LT_PRED(_Pred, _First[_Idx], _First[_Idx-1])) {

```

Comment:
We conventionally put spaces around subtraction operators, and this is required to pass clang-format validation.